### DISCUSSION TITLE
Algolia Index cloud fn error: Error: Cannot find module './algoliaIndex'

### DISCUSSION THREAD

**Discussion by runnedrun** [AntlerVC/firetable#811] [Support / Q&A] [answered]
**Describe the bug**
I enabled the algolia index cloud fn, added my algolia secrets to google cloud secret manager, and then triggered the function by duplicating a row. Then when I looked into my cloud fn logs I saw: Error: Cannot find module './algoliaIndex'

**To Reproduce**
1. Enable Algolia Index extension
2. Add algolia key as detailed here: https://docs.rowy.io/extensions/algolia-index
3. Activate the algolia index cloud fn from the rowy dash
4. Duplicate a row in the table
5. Check cloud fn and see error.

**Expected behavior**
I suppose I expected something to be written to an algolia index.

**Screenshots**
<img width="904" alt="image" src="https://user-images.githubusercontent.com/575530/186430236-f722f1a0-6bee-40b3-b776-d073f3c630dd.png">

**Rowy Run version**
Rowy Run v1.6.1 (thought originally it just said Rowy Run v, then I pressed check for updates, another small bug)

**Desktop (please complete the following information):**
 - OS: Mac 12.4
 - Browser Chrome
 - Version Latest

**Comment by harinij** [comment, marked as answer]
@runnedrun apologies for the delayed reply on this. We have improved the deploying process in the recent releases and this issue shouldn't be happening anymore. We would love for you to give this a try again 🙏 

Also, if you prefer, we can hop on a quick call to walk you through it and get any issues resolves - email us support@rowy.io
